Src/GoogleApis.Core/Apis/Util/PageStreamer.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
In the library we add a different header at the top of each file, take a look at:
https://github.com/google/google-api-dotnet-client/blob/master/Src/GoogleApis/Apis/Requests/BatchRequest.cs
(Remember to change the year to 2015, and don't forget to do the same for the Test file)